synapse/cortex.py

```diff
@@ -4384,6 +4391,9 @@ async def addLayer(self, ldef=None, nexs=True):
 
         s_layer.reqValidLdef(ldef)
 
+        if ldef.get('readonly'):
```

Comment:
If we were to just create the LMDB file when added, i don't think we'd need this?